### PR TITLE
Puppet implementation of property configurator

### PR DESCRIPTION
**Pull request by annakocheshkova**
Implemented Transmission Screen and got rid of Custom Properties screen

### CODE REVIEW

File: Assets/Puppet/PuppetTransmission.cs [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+        }
+        else
+        {
+            childTransmissionTarget.TrackEventWithProperties(EventName.text, GetProperties());
+        }
+    }
+
+    private void OverrideProperties(TransmissionTarget transmissionTarget) 
+    {
+        string overridenAppName = AppName.text;
```
Comment (max-mironov) [inline Assets/Puppet/PuppetTransmission.cs]: Var please?

**Review by max-mironov — APPROVED** [review]
👍

File: Assets/Puppet/PuppetTransmission.cs [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+        childTransmissionTarget.IsEnabledAsync().ContinueWith(task =>
+        {
+            ChildTransmissionEnabled.isOn = task.Result;
+        });
+        TransmissionTarget.text = TransmissionTargetToken;
+        ChildTransmissionTarget.text = ChildTransmissionTargetToken;
+    }
+
+    private string ResolveToken() {
+        if (string.IsNullOrEmpty(TransmissionTarget.text)) {
```
Comment (MatkovIvan) [inline Assets/Puppet/PuppetTransmission.cs:46]: Please format

File: Assets/Puppet/PuppetTransmission.cs [hunk 3]
```diff
@@ -0,0 +1,159 @@
...
+        var childTransmissionTarget = transmissionTarget.GetTransmissionTarget(ResolveChildToken());
+        OverrideProperties(childTransmissionTarget);
+        Dictionary<string, string> properties = GetProperties();
+        if (properties == null)
+        {
+            childTransmissionTarget.TrackEvent(EventName.text);
+        }
+        else
+        {
+            childTransmissionTarget.TrackEventWithProperties(EventName.text, GetProperties());
```
Comment (MatkovIvan) [inline Assets/Puppet/PuppetTransmission.cs:112]: Use `properties` var

File: Assets/Puppet/PuppetTransmission.cs [hunk 4]
```diff
@@ -0,0 +1,159 @@
...
+    public InputField EventName;
+    public InputField AppName;
+    public InputField AppVersion;
+    public InputField AppLocale;
+    public InputField TransmissionTarget;
+    public InputField ChildTransmissionTarget;
+    public GameObject EventProperty;
+    public RectTransform EventPropertiesList;
+    private string TransmissionTargetToken = "";
+    private string ChildTransmissionTargetToken = "";
```
Comment (MatkovIvan) [inline Assets/Puppet/PuppetTransmission.cs:25]: `_` prefix for private vars

**Pull request merged**